clis/teliod/src/main.rs

```diff
@@ -115,14 +161,193 @@ impl CommandListener {
     }
 }
 
+async fn init_api(auth_token: &str) -> Result<DeviceIdentity, TeliodError> {
+    let mut identity_path = dirs::data_local_dir().ok_or(TeliodError::NoDataLocalDir)?;
+    identity_path.push("teliod");
+    if !identity_path.exists() {
+        let _ = fs::create_dir_all(&identity_path);
+    }
+    identity_path.push(&format!("utun10.json"));
+
+    let mut device_identity = DeviceIdentity::new(&identity_path).await;
+
+    let backoff_bounds = ExponentialBackoffBounds {
+        initial: Duration::from_secs(1),
+        maximal: Some(Duration::from_secs(300)),
+    };
+    let mut backoff = ExponentialBackoff::new(backoff_bounds)?;
```

Comment:
I'm a bit puzzled by this, why is this `mut`, but we `.clone()` it subsequently, is the backoff state shared `Arc`?

Reply:
This is `mut` because this `backoff` is used [here](https://github.com/NordSecurity/libtelio/pull/853/files/4d995b85a8ebe7432a72a5ed2649eeec57929151#diff-adacd55ecee4795f10ea1a14eea3906b343e77c9c9d8a717761e1f82f2f2e7faR228)